cove_bods/templates/cove_bods/explore.html

```diff
@@ -14,7 +14,7 @@ <h4 class="panel-title">
                     <span class="glyphicon glyphicon-collapse-up"></span>{% trans 'Download Data' %}
                   </h4>
                 </div>
-                <div id="downloadData" class="collapse in">
+                <div id="downloadData" class="collapse in box-padding">
```

Comment:
I think the more bootstrap thing to do is to have `<div class="panel-body">`. (This is what 360 CoVE does anyway https://dataquality.threesixtygiving.org/data/62eabd80-1d1a-46a4-9e1b-b7820eb1f1fd).